jvm/src/main/scala/fr/hmil/roshttp/HttpDriver.scala

```diff
@@ -6,6 +6,7 @@ import java.nio.ByteBuffer
 import fr.hmil.roshttp.exceptions._
 import fr.hmil.roshttp.response.{HttpResponse, HttpResponseFactory, HttpResponseHeader}
 import fr.hmil.roshttp.util.HeaderMap
+import javax.net.ssl.HttpsURLConnection
```

Comment:
Just checking, how portable is this?
I'm not 100% aware of the limitations of the `javax` package.

Reply:
What do you mean by portable?